doc/README.md

```diff
@@ -39,11 +39,13 @@
             * [.bytesRead](#BlockMap.ReadStream+bytesRead) : <code>Number</code>
             * [.position](#BlockMap.ReadStream+position) : <code>Number</code>
         * [.parse](#BlockMap.parse) ⇒ <code>[BlockMap](#BlockMap)</code>
+        * [.stringify](#BlockMap.stringify) ⇒ <code>String</code>
         * [.versions](#BlockMap.versions) : <code>Array</code>
         * [.create(options)](#BlockMap.create) ⇒ <code>[BlockMap](#BlockMap)</code>
         * [.fromJSON(data)](#BlockMap.fromJSON) ⇒ <code>[BlockMap](#BlockMap)</code>
         * [.createReadStream(filename, blockMap, options)](#BlockMap.createReadStream) ⇒ <code>[ReadStream](#BlockMap.ReadStream)</code>
         * [.parse(value, blockMap)](#BlockMap.parse) ⇒ <code>[BlockMap](#BlockMap)</code>
+        * [.stringify(blockMap, [options])](#BlockMap.stringify) ⇒ <code>String</code>
```

Comment:
Huh?! How come `BlockMap.stringify` has been documented twice? Was one of them supposed to be documenting `.toString()`?

Also, what is the `options` for? Doesn't seem to actually be used? (Or is it there "for future purposes"?)